common/utils/works.js

```diff
@@ -214,11 +214,25 @@ export function getWorkTypeIcon(work: Work): ?string {
 }
 
 export function getLocationType(work: Work, locationType: string) {
+  // TODO rename?
```

Comment:
Yea, I would, it's not what this function is doing.

Reply:
going with `getItemAtLocation`